SwiftStaticLibrarySample/ViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  ViewController.swift
+//  SwiftStaticLibrarySample
+//
+//  Created by Chad Landis on 8/7/17.
+//  Copyright © 2017 Chad Landis. All rights reserved.
+//
+
+import UIKit
+
+// Import Dynamic framework, this is totally normal.  We can access both Swift and Objective-C code
+// that is exported from the module (all Objective-C code in the module header, and all public Swift)
+import SwiftDynamicFramework
+
+// Import Static library, this is totally normal.  We can access both Swift and Objective-C code
+// that is exported from the module (all Objective-C code in the module header, and all public Swift)
+// However, this import statement generates the following warning:
+// "Implicit import of bridging header 'SwiftStaticLibrary.h' via module 'SwiftStaticLibrary' is deprecated and will be removed in a later version of Swift"`"
+// I am using SwiftStaticLibrary.h as a sort of umbrella/module header for the entire library.
+// All Objective-C headers will reside in here.  This is also used inside the target as a bridging
+// header to expose Objective-C code to Swift (the same technique is used for dynamic frameworks).
+// I'm not sure if this warning is spurious (https://bugs.swift.org/browse/SR-3801), or I am doing something wrong here.
+// I don't get the same warning when I import the library into the test target.
+import SwiftStaticLibrary
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        printStaticFromSwift()
+        printDynamicFromSwift()
+
+        ObjcAppCode.printStaticFromObjectiveC()
+        ObjcAppCode.printDynamicFromObjectiveC()
+    }
+
+    func printStaticFromSwift() {
+        print("BEGIN STATIC")
+
+        let a = SampleSwiftStructFromStaticLibrary(name: "static")
+        print("a.fooMethod() = \(a.fooMethod())")
+        print("a.name = \(a.name)")
+
+        let b = SampleObjcClassFromStaticLibrary()
+        print("b.stringMethod1() = \(b.stringMethod1())")
+        print("b.stringMethod2() = \(b.stringMethod2())")
+
+        let c = SwiftStaticLibrary.SampleSwiftClassDefinedInEachLibrary()
```

Comment:
Both the static and dynamic libraries contain a class named `SampleSwiftClassDefinedInEachLibrary`.  The fact that I can use both `SwiftStaticLibrary.SampleSwiftClassDefinedInEachLibrary` and `SwiftDynamicFramework.SampleSwiftClassDefinedInEachLibrary` makes me think that `SwiftStaticLibrary` is in fact a module that I can import (I'm getting Swift namespacing at least).  I'm not sure that is in fact correct.